Podfile

```diff
@@ -0,0 +1,10 @@
+platform :macos, '10.11'
+use_frameworks!
+
+target 'Brooklyn' do
+  pod 'Sparkle'
+end
+
+target 'Canvas' do
+  pod 'Sparkle'
```

Comment:
Do we need this for Canvas?

Reply:
Nope, I was a bit unsure about what all Canvas does.